aten/src/ATen/NestedTensorImpl.cpp

```diff
@@ -177,7 +177,9 @@ NestedTensorImpl::NestedTensorImpl(
   C10_LOG_API_USAGE_ONCE("torch.NestedTensor");
   TORCH_WARN_ONCE(
       "The PyTorch API of nested tensors is in prototype stage and will change "
-      "in the near future.");
+      "in the near future. We recommend specifying layout=torch.jagged when constructing "
+      "a nested tensor, as this layout receives active development, has better operator "
```

Comment:
wow, NJT has better ops coverage now?

Reply:
I think you'd be hard-pressed to find an op that NST supports but NJT doesn't at this point :)